<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinishDialog</class>
 <widget class="QDialog" name="FinishDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Завершение теплового контроля</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="FinishFrame">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>400</width>
     <height>325</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="FinishTitle">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>40</y>
      <width>336</width>
      <height>72</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24px; font-weight:600; color:#252525;&quot;&gt;Завершить тепловой контроль?&lt;br/&gt;&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:14px; color:#252525;&quot;&gt;Проверьте путь сохранения&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FinishNoButton">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>265</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Нет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FinishYesButton">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>265</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Да</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FinishChangePathButton">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>200</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Изменить путь</string>
    </property>
   </widget>
   <widget class="QLabel" name="FinishPathLabel">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>125</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Путь сохранения файлов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FinishPathLineEdit">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>160</y>
      <width>336</width>
      <height>32</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
